bin/bitbake-setup

```diff
@@ -61,6 +62,12 @@ def write_upstream_config(config_dir, config_data):
     with open(os.path.join(config_dir, "config-upstream.json"),'w') as s:
         json.dump(config_data, s, sort_keys=True, indent=4)
 
+def write_sources_state(config_dir, config_data):
+    sources = {}
+    sources['sources'] = config_data
+    with open(os.path.join(config_dir, "sources-state.json"),'w') as s:
```

Comment:
I think this (and the function name) should be named 'sources-fixed-revisions.json', which is more clear.